msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -58,79 +58,467 @@ For more information, see [public developer preview app manifest schema](../../r
+                        },
+                        type: 'auth',
+                    };
+                }
+                break;
+            case 'message':
+                {
+                    const cardJson = {
+                        type: 'AdaptiveCard',
+                        version: '1.4',
+ 
+                        body: [
+                            {
+                                type: 'TextBlock',
+                                text: 'Bot Config Fetch',
+                            },
+                        ],
+                    };
+                    const card = CardFactory.adaptiveCard(cardJson);
+ 
+                    response = {
+                        config: {
+                            value: {
+                                height: 200,
+                                width: 200,
+                                title: 'test card fetch',
+                                card,
+                            },
+                            type: 'continue',
+                        },
+                    };
+                }
+                break;
+            default:
+                console.log('no default');
+        }
+ 
+        return response;
+    }
+ 
+    async handleTeamsConfigSubmit(_context, configData) {
+        let response = {};
+        switch (configData.command) {
+            case 'card':
+                {
+                    const cardJson = {
+                        type: 'AdaptiveCard',
+                        version: '1.4',
+ 
+                        body: [
+                            {
+                                type: 'TextBlock',
+                                text: 'Bot Config Submit',
+                            },
+                        ],
+                    };
+                    const card = CardFactory.adaptiveCard(cardJson);
+ 
+                    response = {
+                        config: {
+                            value: {
+                                height: 200,
+                                width: 200,
+                                title: 'test card submit',
+                                card,
+                            },
+                            type: 'continue',
+                        },
+                    };
+                }
+                break;
+            case 'message':
+                {
+                    response = {
+                        config: {
+                            value: 'config submit text',
+                            type: 'message',
+                        },
+                    };
+                }
+                break;
+            default:
+                console.log('no default');
+                break;
+        }
+ 
+        return response;
+    }
+```
+
+---
+
+#### C# code snippets
+
+The following code snippets shows an example of `OnTeamsConfigFetchAsync`, `OnTeamsConfigSubmitAsync` and `OnInvokeActivityAsync`:
+
+# [C# 1](#tab/teams-bot-sdk)
```

Comment:
Can we switch C#1 and C#2, then we make C#1 to be onInvoke section, which aligns the order of JS (like JS #1 is onInvoke too)

Reply:
Updated.